### ISSUE TITLE
.text is empty on Soup creation

### ISSUE THREAD

**Issue by maxhumber** [maxhumber/gazpacho#28]
**Describe the bug**

When I create a soup object...

**To Reproduce**

Calling .text returns an empty string:

```python
from gazpacho import Soup

html = """<p>&pound;682m</p>"""

soup = Soup(html)
print(soup.text)
''
```

**Expected behavior**

Should output:

```python
print(soup.text)
'£682m'
```

**Environment:**
 - OS: macOS
 - Version: 1.1

**Additional context**

Inspired by this [S/O question](https://stackoverflow.com/questions/2087370/decode-html-entities-in-python-string)

**Comment by DeepanshuGOYAL05** [comment]
Can you assign this to me?

**Comment by prashant314** [comment]
Hi can you please assign this to me?

**Comment by DeepanshuGOYAL05** [comment]
I have worked on beautifulsoup a lot, you can check my repo?

**Comment by maxhumber** [comment]
I'll assign to @DeepanshuGOYAL05 because he asked first! 🙃

**Comment by DeepanshuGOYAL05** [comment]
Thanks!!, can I do it separately or in the same file?

**Comment by maxhumber** [comment]
Probably in the same file. It should only be like a couple lines of code

**Comment by DeepanshuGOYAL05** [comment]
yaa, okay!!
Basically you want inner html value of a tag

**Comment by maxhumber** [comment]
gazpacho is already able to find inner tag text values! The problem is exclusively on object creation (the fix is I think far simpler than you're imagining!)

**Comment by DeepanshuGOYAL05** [comment]
Ohh Okay!!

**Comment by DeepanshuGOYAL05** [comment]
Hey, I am able to get the output. Can i make a separate file for that?

**Comment by maxhumber** [comment]
... I don't think you need a separate file

**Comment by DeepanshuGOYAL05** [comment]
Please check it at your end

**Comment by maxhumber** [comment]
Thanks for all your hard work @DeepanshuGOYAL05. But I think there's a severe misunderstanding in what is required to fix this issue. It's not a matter of using `Soup.find()` to solve the problem (as detailed in the documentation). Nor, is this Issue looking for an ad-hoc script that specifies a fix for this exact case. The Issue is about changing `Soup(html).text` to immediately work after instantiation without having to use `find`.

**Comment by jeffsieu** [comment]
Hi, can I have a go at this?

**Comment by maxhumber** [comment]
Fixed by #29
